### ISSUE TITLE
Uncaught TypeError: Cannot read property '0' of null

### ISSUE THREAD

**Issue by compojoom** [flipbyte/yup-schema#8]
I'm having troubles with this validation schema:

```
          ['object'],
          ['shape', {
              url: [['string'], ['required']],
              path: [['string'], ['required']]
            }],
          ['nullable']
          ['default', null]

```

If I remove the default null, then it works fine. But I need the field to have a default value of null

**Comment by compojoom** [comment]
Actually, I have a typo in the schema above. I'm missing the comma after ['nullable']. Adding the comma however triggers another error:
Uncaught TypeError: Cannot read property 'constructor' of null

This seems to happen in the processArgs function. Since my argument is not undefined, but null and typeof reports null as an object, but null.constructor doesn't exist...

**Comment by compojoom** [comment]
@easeq would you please review my pull request, merge and release a new version if it doesn't break something else :)

**Comment by easeq** [comment]
Released version 0.1.5
